docs/specviz/redshift.rst

```diff
@@ -13,8 +13,8 @@ From the notebook
 
 The range of the slider, as well as the resolution of a single
 step in the slider, can be set from a notebook cell using the 
-:meth:`jdaviz.configs.specviz.Specviz.set_redshift_slider_bounds` method by
-specifying the ``range`` and/or ``step`` keywords.  By setting either or both
+:meth:`jdaviz.configs.default.plugins.line_lists.line_list_mixin.LineListMixin.set_redshift_slider_bounds`
+method by specifying the ``range`` and/or ``step`` keywords.  By setting either or both
```

Comment:
It is less clear now that you are supposed to access this method in Specviz, hence the suggestion.
```suggestion
:meth:`~jdaviz.configs.default.plugins.line_lists.line_list_mixin.LineListMixin.set_redshift_slider_bounds`
method in ``specviz`` by specifying the ``range`` and/or ``step`` keywords.  By setting either or both
```